core/trino-main/src/main/java/io/trino/metadata/SystemSecurityMetadata.java

```diff
@@ -184,6 +184,16 @@ default void validateEntityKindAndPrivileges(Session session, String entityKind,
      */
     Optional<Identity> getFunctionRunAsIdentity(Session session, CatalogSchemaFunctionName functionName);
 
+    /**
+     * Set the owner of the specified function
+     */
+    void setFunctionOwner(Session session, CatalogSchemaFunctionName function, TrinoPrincipal principal);
```

Comment:
Is there an operation which this method should be used by? There's not `ALTER FUNCTION .. SET AUTHORIZATION` for functions so I don't think there is such operation.

Reply:
There is no none, but it was suggested by @ksobolew and @kokosing to keep the conversion, also it is useful for tests ;)